cfme/web_ui/mixins.py

```diff
@@ -49,7 +49,7 @@ def get_tags(tag="My Company Tags"):
     tags = []
     for row in sel.elements(
             "//*[(self::th or self::td) and normalize-space(.)={}]/../.."
-            "//td[img[contains(@src, 'smarttag')]]".format(
+            "//td[not(contains(@class, 'label'))]".format(
```

Comment:
this was more general solution, but now I see in cloud providers it will fail, changing